database/Listing.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
```

Comment:
Thanks for the feedback! In this case, I'll be keeping my database's List and Listing models separate so that I can create a unified syntax around seeding entries, e.g: 
```
List.Seed()
  .then(Listing.Seed())
```